tests/fixtures_regress/tests.py

```diff
@@ -198,6 +211,7 @@ def test_unknown_format(self):
                 verbosity=0,
             )
 
+    @skipIfNonASCIIPath
```

Comment:
I think this is a bug/limitation in Python 2 due to the line: `if not expected_regexp.search(str(exc_value)):` in `unittest/case.py`.